pyglmnet/pyglmnet.py

```diff
@@ -198,33 +234,26 @@ def fit(self, X, y):
                 fit_params[-1]['beta'] = fit_params[-2]['beta']
 
             # Iterate until convergence
-            no_convergence = 1
             threshold = self.threshold
             alpha = self.alpha
 
-            t = 0
-
             # Initialize parameters
             beta = np.zeros([p + 1, k])
             beta[0] = fit_params[-1]['beta0']
             beta[1:] = fit_params[-1]['beta']
 
             g = np.zeros([p + 1, k])
-            # Initialize cost
-            L = []
-            DL = []
```

Comment:
I replaced this with `DL = list()` because it's very easy to confuse `[]` and `{}` -- better to spell it out.